code/lora/lora.py

```diff
@@ -0,0 +1,298 @@
+##############################
+# WORKS ONLY WITH PYTHON 3.7 #
+##############################
+
+import os
+import time
+#TODO import argparse
+from pathlib import Path
+from SX127x.LoRa import *
+from SX127x.board_config import BOARD
+
+HOME_PATH = Path.home()
+LORA_LOGS = HOME_PATH / 'lora_logs'
```

Comment:
The `/` operator from pathlib is hilarious :D

Reply:
Do you think that these paths should be configparsed?

Reply:
I see no reason why not to, configparser provides interpolation so it is achievable, go for it!